kili/cli.py

```diff
@@ -73,13 +73,18 @@ def list_project(api_key: str,
 
     """
     kili = Kili(api_key=api_key, api_endpoint=endpoint)
-    projects = kili.projects(fields=['title', 'id', 'description', 'numberOfAssets',
-                             'numberOfRemainingAssets', 'numberOfReviewedAssets'],
-                             first=first,
-                             disable_tqdm=True)
+    projects = cast(
+        List[Dict],
+        kili.projects(
+            fields=[
+                'title', 'id', 'description', 'numberOfAssets',
+                'numberOfRemainingAssets', 'numberOfReviewedAssets'],
+            first=first,
+            disable_tqdm=True))
     projects = pd.DataFrame(projects)
-    projects['progress'] = round(
-        (1 - projects['numberOfRemainingAssets'] / projects['numberOfAssets']) * 100, 1)
+    projects['progress'] = projects.apply(lambda x: round(
+        (1 - x['numberOfRemainingAssets'] / x['numberOfAssets'])* 100 , 1)
+        if x['numberOfAssets'] != 0 else 0, axis=1)
 
     # Add '%' to PROGRESS if PROGRESS is not nan
     projects['PROGRESS'] = [(str(progress) + '%') if progress >=
```

Comment:
Now that we put 0 if x['numberOfAssets'] == 0, we don't have to test if progress >=0 anymore